theme/palette/elements.less

```diff
@@ -7,3 +7,4 @@
 @sk-footer: @sk-mid-gray-light;
 @sk-background: @sk-gray-light;
 @sk-yellow: #fffce3;
+@sk-gold: #ffc600;
```

Comment:
@stephaniemoss do we use that light yellow colour (#fffce3) anywhere else? If not, do we plan to? Wondering whether this even belongs in the canonical list of colours, or whether it's actually just a one-off that we're trying to move away from.